### PR TITLE
Add agreement and submit-button

### PR DESCRIPTION
**Pull request by geckotang**
- First you confirm agreement and click the checkbox.
- And then you can click submit button.
- Finally you click submit button after that a message is displayed on your console.

### CODE REVIEW

**Comment by geckotang** [comment]
@watilde Please make sure.

File: lib/components/submit-button.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  },
+  componentDidMount() {
+    componentHandler.upgradeDom()
+  },
+  _onSubmit() {
+    this.props.onSubmit()
+  },
+  render() {
+    return (
+      <button onClick={this._onSubmit} type="button" className="mdl-button mdl-js-button mdl-button--raised mdl-button--colored" disabled={this.props.isAgreed ? false : true}>
```
Comment (watilde) [inline lib/components/submit-button.js:20]: `onClick` is here! It's ok at the moment, but we will try to move it to dispatcher.

Comment (geckotang) [inline lib/components/submit-button.js:20]: @watilde Hmm, do you want to use `addEventListener` ?  Like this:

``` javascript
componentDidMount() {
  buttonEl.addEventListener('click', this._onSubmit)
  componentHandler.upgradeDom()
},
componentWillUnmount() {
  buttonEl.removeEventListener('click', this._onSubmit)
}
```

Is this way common practice in React?

Comment (watilde) [inline lib/components/submit-button.js:20]: Yap I meant it! https://facebook.github.io/react/tips/dom-event-listeners.html

Comment (geckotang) [inline lib/components/submit-button.js:20]: I see. However I guess it means that you want to attach DOM events to other element like `window` in a component.

Comment (watilde) [inline lib/components/submit-button.js:20]: Ahh you are correct again. Let me put `+1` on here.

**Comment by watilde** [comment]
Dankjewel :smiley_cat:

**Pull request merged**